tools/bazel

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+set -e
+
+default_bazel_version='0.28.1'
+
+if [ "$BUILDKITE" = true ]; then
+    bazel_version='host'
+else
+    case $(uname -s) in
+    Darwin|Linux)
+	# shellcheck disable=SC2153
+	if [ -z "$BAZEL_VERSION" ]; then
+	    bazel_version="$default_bazel_version"
+	else
+	    bazel_version="$BAZEL_VERSION"
+	fi
+        ;;
+    *)
+	# windows, presumably
+	bazel_version='host'
+	;;
+    esac
+fi
+
+case "$bazel_version" in
+    'host')
+        bazel_version=$("$BAZEL_REAL" version | awk '/Build label/ {print $3}' | cut -d '-' -f 1)
+        bazel="$BAZEL_REAL"
+        ;;
+    '0.28.1')
```

Comment:
We used to be able to support several versions of bazel with the matrix. How can we do this here? We like to test on the oldest we can support and the newest (although sometimes that is a single version).

Reply:
I'll add some notes. The TLDR is that a build matrix across bazel versions is very well supported with this script.